wp-content/themes/haruoshi_origin/src/scss/style.scss

```
@@ -4,6 +4,7 @@
 html {
 	font-family: Arial;
 	color: $main-color;
+	font-size: 14.4px;
```

地獄みたいなフォントサイズの指定やめてください
やるならせめて
```suggestion
	font-size: 14px;
```
とか

---

だってデザインがそうなってたから笑

---

done https://github.com/oooshima/origin_wordpress/pull/25/commits/39c4fbd57abb198238b60fa2ea29ccca6e89ddfa